src/CMakeLists.txt

```diff
@@ -37,10 +37,6 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
   list(APPEND boost_components zlib)
 endif()
 find_package(Boost 1.51 REQUIRED COMPONENTS ${boost_components})
-if(${Boost_MAJOR_VERSION}.${Boost_MINOR_VERSION} VERSION_LESS 1.59)
```

Comment:
we dont need threadsafe ptree anymore and so we dont need boost::thread

Reply:
yes!